pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -150,13 +149,8 @@ func Command(run *params.Run, ioStreams *cli.IOStreams) *cobra.Command {
 				}
 			}
 
-			pacInfo, err := info.GetPACInfo(ctx, run, opts.targetNamespace)
```

Comment:
do you need to remove it ? i think we need both, check for pac info and then check for the secret installed....

Reply:
If we are checking presence of secret then i don't think we need PACInfo because here we were using Provider key from pacInfo to check its empty or its setted to `GithubApp` but as we are relying on secret presence  now we no more required that info right

Reply:
humm interesting, yeah we definitvely should have some... the same as what we have on configmap like part-of

Reply:
i haven't checked the full code but don't we get the route information from there? would that break?

Reply:
No that wont break because route information we get here https://github.com/openshift-pipelines/pipelines-as-code/blob/main/pkg/cli/webhook/webhook.go#L60 and i have not updated anything related to this and for 
tkn-pac create repo and tkn-pac bootstrap we don't require route information